### ISSUE TITLE
Advice on scaling the viz on smaller devices

### ISSUE THREAD

**Issue by dagda1** [patternfly/react-topology#132]
**Describe the problem**

I have increased the height and width of the pipeline node model. 

```ts
const task: PipelineNodeModel = {
  width: NEW_TASK_WIDTH,
  height: NEW_TASK_HEIGHT,
```

Now, the nodes do not fit on the screen on smaller resolutions:

<img width="1090" alt="n
ode" src="https://github.com/patternfly/react-topology/assets/118328/c4912933-95b8-4b4e-a73e-99cc6a9ee77f">

As I am relatively new to this package, I don't know what my options are in this scenario.

I think it would be good to have a vertical layout for mobiles but all I can find is the `PipelineDagreLayout`.

Are there any other layouts or are there any docs about creating your own layout?

**Comment by dagda1** [comment]
TLDR; of the above post would be...

Can I display a`PipelineDagreLayout` in stacked form or is horizontal the only option.  If not, is there any guidance on creating vertical `PipelineDagreLayout` layout
